installer/templates/static/brunch/package.json

```diff
@@ -6,8 +6,8 @@
     "watch": "brunch watch --stdin"
   },
   "dependencies": {
-    "phoenix": "file:<%= brunch_deps_prefix %><%= phoenix_path %>"<%= if html do %>,
-    "phoenix_html": "file:<%= brunch_deps_prefix %>deps/phoenix_html"<% end %>
+    "phoenix": "~<%= phoenix_version %>"<%= if html do %>,
+    "phoenix_html": "~<%= phoenix_version %>"<% end %>
```

Comment:
I don't think we should depend on the NPM package by default. Using the one inside deps/ is the simplest approach because developers just update Phoenix. If we put them apart, developers will now need to update Phoenix, then Phoenix may be incompatible with their Phoenix NPM package, then they need to update the package, etc. I think we should still depend on "file:deps/phoenix" and leave the NPM package for those who care about not having a file dependency.